samples/symNotepad.c

```diff
@@ -0,0 +1,199 @@
+         }
+         else if( c == 0x13 ) {        // Save
+            puts( "\n========================= Save =========================" );
+            puts( "\nPress any key to save." );
+            c = getchar();
+            for( l = 0; l <= p; l++ ) {
+               tapio[l] = buffer[l];
+            }
+            l++;
+            tapio[l] = 0x00;
+            puts( "Saving to tape." );
+            error = dumpt( 'N', (int) tapio, (int) tapio+p );
+            if( error ) {
+               puts( "\nTape error." );
+            }
+            else
+            {
+               putchar( '\r' );
+               for( l = 0; l < 25; l++ ) {
+                  putchar( '\n' );
+               }
+            }
+            puts( "===================== Sym-1 Notepad ====================\n" );
+            for( l = 0; l <= p; l++ ) {
+               putchar( buffer[l] );
+               if( buffer[l] == '\r' ) {
+                  putchar( '\n' );
+               }
+            }
+         }
+         else if( c == 0x0C ) {        // Load
+            p = 0;
+            puts( "\nLoading from tape." );
+            memset( buffer, 0, heap_size );
+            memset( tapio, 0, TAPIO_MAX_SIZE );
+            error = loadt( 'N' );
+            if( error ) {
+               puts( "\nTape error." );
+               puts( "===================== Sym-1 Notepad ====================\n" );
+            }
+            else
+            {
+               for( l = 0; l <= heap_size; l++ ) {
+                  buffer[l] = tapio[l];
+               }
+
+	       p = strlen( buffer );
+
+               putchar( '\r' );
+               for( l = 0; l < 25; l++ ) {
+                  putchar( '\n' );
+               }
+               puts( "===================== Sym-1 Notepad ====================\n" );
+
+               for( l = 0; l <= p; l++ ) {
+                  putchar( buffer[l] );
+                  if( buffer[l] == '\r' ) {
+                     putchar( '\n' );
+                  }
+               }
+            }
+         }
+         else if( c == 0x03 ) {        // Clear
+            p = 0;
+            memset( buffer, 0, heap_size );
+            putchar( '\r' );
+            for( l = 0; l < 25; l++ ) {
+               putchar( '\n' );
+            }
+            puts( "===================== Sym-1 Notepad ====================\n" );
+         }
+         else if( c == 0x18 ) {        // Exit
+            writing = 0;
+            running = 0;
+         }
+         else {
+            if( p >= heap_size - 1 ) {
+               puts( "\n========================= End  =========================" );
+               puts( "Buffer full." );
+            }
+            else {
+               if( c == '\r' ) {
+                  putchar( '\n' );
+               }
+               buffer[p] = c;
+               putchar( c );
+            }
+            p++;
+         }
+      }
+   }
+
+   free( buffer );
+
+   puts( "\nEnjoy your day!\n" );
+
+   return;
+}
+
```

Comment:
Please change this code to adhere to the cc65 C code style rules - see https://github.com/cc65/wiki/wiki/C-and-Assembly-sources-style-guide

Reply:
**DONE**.  Please check and confirm.